app/models/vm_or_template.rb

```diff
@@ -1092,7 +1092,7 @@ def refresh_ems_sync
   end
 
   def self.post_refresh_ems(ems_id, update_start_time)
-    update_start_time = update_start_time.utc
+    update_start_time = update_start_time.utc - 1.minute
```

Comment:
This seems really hacky, lets see if there's a better way to do this.

Reply:
I am open for suggestions. We use time to determine which the vm was created so I wonder what we can do here.

Reply:
Yeah I'm thinking if we could try to get update_start_time from when refresh_new_target started, haven't checked if that is feasible yet

Reply:
@agrare @pkliczewski any updates on your side?

Reply:
@pkliczewski could you just call `#post_create_actions_queue` from `refresh_new_target` if the vm was created instead of messing with the ems_refresh_start_time?

Reply:
@agrare will try